test/XRP/web-socket-network-client.test.ts

```diff
@@ -27,20 +30,28 @@ const webSocketNetworkClient = new WebSocketNetworkClient(
   console.log,
 )
 
+const grpcNetworkClient = new GrpcNetworkClient(rippledGrpcUrl)
+
 function sleep(ms: number) {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
 
 describe('WebSocket Tests', function (): void {
   // Retry integration tests on failure.
-  this.retries(3)
+  //this.retries(3)
```

Comment:
sorry something I also left.... we could probably get away with removing this too.  this was a relic from when we weren't generating new wallets for new test runs, and simultaneous CI runs were creating sequence number clashes that created occasional failures.